src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -214,9 +214,9 @@ class BidValidationSpec extends BaseSpec {
     }
 
     def "PBS should return an error when GVL Id alias refers to unknown bidder alias"() {
-        given: "Default basic BidRequest"
+        given: "Default basic BidRequest with aliasgvlids and aliases"
         def bidderName = PBSUtils.randomString
-        def validId = 1;
+        def validId = PBSUtils.getRandomNumber(1)
```

Comment:
Well, take a look at a boundary value analysis and equivalence partitioning testing.
'validId = 1' was good here as it is on the bound.
With a random number from getRandomNumber() you don't verify that just exactly after 0 integer PBS processes GVL Id alias in a different way.